recipes/python-ghostscript/win_version.patch

```diff
@@ -0,0 +1,11 @@
+--- python-ghostscript-v0.7.orig/setup.cfg	2023-12-20 13:18:30.452497083 -0600
++++ python-ghostscript-v0.7/setup.cfg	2023-12-20 13:18:40.747546844 -0600
+@@ -22,7 +22,7 @@
+ 
+ [metadata]
+ name = ghostscript
+-version = attr:ghostscript.__version__
++version = 0.7
```

Comment:
Just curious, why do you need this patch? What went wrong here?

Reply:
On windows, setuptools is failing to evaluate the version in this manner.  I needed it to make CI green here, but perhaps it would not be necessary within a feedstock building on linux64 for noarch

Reply:
Yep, noarch with a failing Windows would be ok. I also would like to see the error, I'm curious what is going on there.

Reply:
Well, I must have been dreaming that I tested this without the hardcoded path patch.  Onward and upward!